clang/test/SemaSYCL/check-notdirect-attribute-propagation.cpp

```diff
@@ -16,5 +16,8 @@ void parallel_for(Type lambda) {
 }
 
 void invoke_foo2() {
+  // CHECK-LABEL:  FunctionDecl {{.*}} invoke_foo2 'void ()'
+  // CHECK:  `-FunctionDecl {{.*}} _ZTSZ11invoke_foo2vE10KernelName 'void ()'
```

Comment:
I would suggest {{.*}}KernelName{{.*}} Differences in name mangling on Windows will cause issues otherwise.

Reply:
I agree with this thought, and changed it per your suggestion. However, please note that there are other lit tests following this name mangled pattern, with which I intended to be consistent.